### PR TITLE
Update example input files

### PR DESCRIPTION
**Pull request by Mikolaj-A-Kowalski**
Should resolve Issue #29. 
- All sample input files use $SCONE_ACE EnvVar
- S(a.b) syntax was updated in the examples
- Name of the S(a,b) file was made consistent with NEA naming convention
- Removed tab and trailing whitespace

### CODE REVIEW

File: InputFiles/c5g7_mox [hunk 1]
```diff
@@ -113,7 +113,7 @@ viz {
 
 nuclearData {
   handles {
-    ce {type aceNeutronDatabase; aceLibrary /home/mak60/myACE/JEF311.aceXS;}
+    ce {type aceNeutronDatabase; aceLibrary $SCONE_ACE;}
   }
   materials {
     numberOfGroups 69;
```
Comment (ChasingNeutrons) [inline InputFiles/c5g7_mox:119]: Maybe we should delete this line since it isn't used in this input.

File: InputFiles/volCalc [hunk 2]
```diff
@@ -1,45 +1,45 @@
-// Ray-tracing calculation of relative volumes of diffrent materials 
-// Used to benchmark performance and debug geometry implementations 
+// Ray-tracing calculation of relative volumes of diffrent materials
```
Comment (ChasingNeutrons) [inline InputFiles/volCalc:1]: Pedantic: 'different'

File: InputFiles/c5g7_mox (hunk 1 above)
Comment (Mikolaj-A-Kowalski) [reply InputFiles/c5g7_mox:119]: 100% agree (encourage though to use GitHub suggestion feature ;-) )

Comment (Mikolaj-A-Kowalski) [inline InputFiles/c5g7_mox:119]:
```suggestion
```

File: InputFiles/volCalc [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
-  universes { 
-    domain { id 1; type rootUniverse; border 1; fill u<2>; } 
-    space  { id 2; type cellUniverse; cells (1 2 3); } 
-    // Veritcal Pin is rotated to align with X-axis 
-    // Translation happens in the rotated frame 
-    pin    { id 3; type pinUniverse; origin (-2.0 0.0 0.0); rotation (90.0 90.0 0.0); radii (1.0 0.0); fills (fuel Al); } 
+  universes {
+    domain { id 1; type rootUniverse; border 1; fill u<2>; }
+    space  { id 2; type cellUniverse; cells (1 2 3); }
+    // Veritcal Pin is rotated to align with X-axis
```
Comment (ChasingNeutrons) [inline InputFiles/volCalc:40]:
```suggestion
    // Vertical Pin is rotated to align with X-axis
```

**Review by ChasingNeutrons — APPROVED** [review]
Looks great! Only tiny things to adjust.

**Comment by Mikolaj-A-Kowalski** [comment]
I will marge then after the checks are done! 
Thanks for the review.

**Pull request merged**